.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -7,6 +7,10 @@
 <!-- IMPORTANT ECLIPSE NOTE: If you change this file, you must restart Eclipse
  for your changes to take effect in its Checkstyle integration. -->
 <suppressions>
+    <!-- Eclipse/Idea APT generated code -->
+    <suppress files="[/\\]generated_src[/\\]" checks="." />
+    <suppress files="[/\\]src[/\\]generated[/\\]java[/\\]" checks="." />
```

Comment:
Sorry, don't understand the feedback -- doesn't `[/\\]generated_src[/\\]"` cover that case? FWIW, these lines were copied directly from the ".baseline" in Foundry, and in Eclipse I confirmed that Checkstyle warnings show up for these files before this change, but don't after the change.